gapic/api/artman_bigtable.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   api_name: bigtable
   api_version: v2
+  package_type: grpc_client
```

Comment:
Unfortunately the common.yaml won't work in this case. Based on the config loading logic, the config is loaded in sequence which means the later loaded ones will override the previous ones. Since this PR is blocking Java GA I suggest we tackle this problem in a follow-up PR. Issue created: https://github.com/googleapis/artman/issues/148